.github/workflows/actions/setup_dev_ubuntu/install_libraries

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+set -x
+
+# Install ghc and cabal
+ghcup install ghc 8.10.7
+ghcup set ghc 8.10.7
```

Comment:
Shouldn't we use `inputs.version_ghc` somehow here?

Reply:
Yes we must. Thank you. Fixed.